servers/src/main/java/tachyon/master/file/meta/FilePersistenceState.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.file.meta;
+
+/**
+ * The persistence state of a file.
+ */
+public enum FilePersistenceState {
+  NOT_PERSISTED, SCHEDULED, PERSISTING, PERSISTED
```

Comment:
what is the different between SCHEDULED and PERSISTING? this class should probably contain documentation that explains the whole asynchronous persistence workflow